src/main/java/com/yscope/logging/log4j1/AbstractBufferedRollingFileAppender.java

```diff
@@ -56,8 +56,12 @@
 public abstract class AbstractBufferedRollingFileAppender extends EnhancedAppenderSkeleton
     implements Flushable
 {
-  protected String currentLogPath = null;
+  // Appender settings, some of which may be set by Log4j through reflection.
+  // For descriptions of the properties, see their setters below.
+  protected String baseName = null;
+  protected String currentLogFilePath = null;
```

Comment:
Can we move this to the clp class since it's not used in this class?

Reply:
done